lib/app/upload.ex

```diff
@@ -4,7 +4,7 @@ defmodule App.Upload do
   """
   import SweetXml
 
-  @region Application.compile_env(:ex_aws, :region)
+  @region Application.get_env(:ex_aws, :region)
```

Comment:
I don't _want_ to do `Application.get_env` ... it's just the only thing that "fixed" #68 ...

Reply:
Removed. (Hard-coded the region; for now ...)